ComcastCRMFrameworkSample/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java

```diff
@@ -0,0 +1,8 @@
+package com.comcast.crm.objectrepositoryutility;
+
+public class CreateProductPage {

```

Comment:
change the xpath
do not hardcode the data

Reply:
fixed